internal/controller/storagescale_controller_test.go

```diff
@@ -51,7 +51,24 @@ var _ = Describe("StorageScale Controller", func() {
 						Name:      resourceName,
 						Namespace: "default",
 					},
-					// TODO(user): Specify other spec details if needed.
+					Spec: scalev1alpha.StorageScaleSpec{
+						IbmCnsaVersion: "v5.2.2.1",
+						MachineConfig: scalev1alpha.MachineConfig{
+							Create: false,
```

Comment:
no need to init default values in go. Bool is initialized as `false`, so you can skip this declaration and also initializing `MachineConfig` a well. When creating a new Spec by default it will contain the default init value of the MachineConfig, Cluster and LocalVolumeDiscovery structures.